src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ImporterLocationService.cs

```diff
@@ -285,6 +284,10 @@ public class ImporterLocationService
                 .And(location => location.ParliamentaryConstituency_Code == (parliamentaryConstituency != null ? parliamentaryConstituency.Code : null)
                                  && location.ParliamentaryConstituency_Name == (parliamentaryConstituency != null ? parliamentaryConstituency.Name : null));
 
+            predicateBuilder = predicateBuilder
+                .And(location => location.PlanningArea_Code == (planningArea != null ? planningArea.Code : null) 
+                                 && location.PlanningArea_Name == (planningArea != null ? planningArea.Name : null));
+
```

Comment:
Yes looks like it was missed when Planning Areas were originally added.

Probably irrelevant to historical data like you say but worth noting that while the 'ignored' feature filtered out **rows** of those ignored levels, it didn't mean the columns associated with those levels weren't necessarily in use.

For example we could ignore 'Region' today to filter out all Regional rows from being imported but an LA or any other type of row that additionally specifies a Region attribute would still have the Region code/name stored against that location.

This bit of query criteria code is used to find a location unique by all location attributes rather than just its principal attribute so it theoretically could have been an issue. It seems unlikely that data for any other level would have used Planning Area as an additional attribute though so not worth investigating.